packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -413,6 +421,14 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
     }
   }, [state.contentSize, state.virtualizer, state.isAnimating, onLoadMore, isLoading]);
 
+  let keysBefore = [];
+  let key = columnState.currentlyResizingColumn;
+  do {
+    keysBefore.push(key);
+    key = tableState.collection.getKeyBefore(key);
+  } while (key != null);
+  let resizerPosition = keysBefore.reduce((acc, key) => acc + columnState.getColumnWidth(key), 0) + (direction === 'ltr' ? -1 * scrollPosition : scrollPosition);
```

Comment:
This is much better now but there is still one iffy case:
1. Navigate to https://reactspectrum.blob.core.windows.net/reactspectrum/0957aabaea66012ab93157805caf52e75f0fc312/storybook/index.html?path=/story/tableview--allowsresizing-uncontrolled-static-widths&providerSwitcher-toastPosition=bottom
2. Resize the size column so it extends past the TableView's right edge (aka just drag it to the right)
3. Trigger resize via the column header. Note that the resizer visually appears far off to the right, but the physical resizer is actually scrolled into view

Happy for this to be followup

Reply:
omg what an edge case, good find. I'll see what I can do about it.

Reply:
The resizer can't go inside the ScrollView so you don't need to accommodate for the scroll position manually? If it could, it would probably be a lot less janky. Also I guess the resizer should be clipped by the scrollview when the resizing column is out of view.